packages/perseus/src/widgets/__tests__/__snapshots__/interactive-graph.test.ts.snap

```diff
@@ -3302,15 +3302,15 @@ exports[`snapshots should render correctly with locked lines 1`] = `
                   >
                     <pattern
                       height="20"
-                      id="cartesian-9"
+                      id="cartesian-31"
```

Comment:
I'm not 100% certain, but I think this snapshot update has to do with [how Mafs does the ID](https://github.com/stevenpetryk/mafs/blob/168edc7816dce6a5402b83d62f5b58ba74c6a9b9/src/display/Coordinates/Cartesian.tsx#L29) for the Cartesian plane. [It uses an incrementer as a hack for unique IDs](https://github.com/stevenpetryk/mafs/blob/168edc7816dce6a5402b83d62f5b58ba74c6a9b9/src/display/Coordinates/Cartesian.tsx#L7). I really don't know why there are 22 more cartesian planes here than there were before though??

I also don't understand why the snapshot even previously only had `cartesian-8` and `cartesian-9`. Not sure where 1-7 went 🤔 

I wonder if @jeremywiebe would know?

Reply:
I suspect that tests that involve the `<Cartesian />` component "burn" ids even if they don't involve capturing a snapshot. As long as the tests (including snapshot tests) run in the same order (which it seems they do) we should be ok with re-numbering these ids. 

It'd be nice to have something like [`useUniqueId()`](https://khan.github.io/wonder-blocks/?path=/docs/packages-core-useuniqueid--docs) from WB, but I don't think it's reasonable to introduce that into Mafs.

Reply:
I think we need to find a fix for this, I keep having to regenerate snapshots.